app/core/config.ts

```diff
@@ -0,0 +1,72 @@
+﻿namespace dogsrus.virtdog {
+  export class DogConfiguration {
```

Comment:
Personally I don't think I would use a class for this. You don't need private values or member functions. You aren't using inheritance or anything you get with classes. I would probably just create an object literal with all the values and export the type for use elsewhere.